solutions/landing-zone-v2/onboarding-admin.md

```diff
@@ -6,9 +6,9 @@
     ```
     cd <LZ_FOLDER>
     ```
-1. Get the hierarchy/admin-sandbox package
+1. Get the hierarchy/admin-experimentation package
     ```
-    kpt pkg get https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit.git/solutions/hierarchy/admin-sandbox@main ./landing-zone/hierarchy/Testing/Admins/<admin name>
+    kpt pkg get https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit.git/solutions/hierarchy/admin-experimentation@main ./landing-zone/hierarchy/Testing/Admins/<admin name>
```

Comment:
hierarchy/Testing/Admins needs to be hierarchy/tests/admins

Reply:
done